### PR TITLE
Avro 1887

### PR DESCRIPTION
**Pull request by spacharya**
I have made the setup to use yetus.
A quick setup of yetus can be done using this script : https://gist.github.com/spacharya/c1013fda8fe5d864fd028e8e5c478b81

You will need to have docker present on that machine.

`/bin/bash ../yetus/precommit/test-patch.sh --plugins=all,-compile,-javadoc --user-plugins=share/pre_commit/   --build-tool=build  https://github.com/apache/avro/pull/191 --docker --dockerFile=share/docker/Dockerfile --dirty-workspace`

Is a sample deployment.

I would appreciate if people can look at the changes and test it and let me know if there are any improvements.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -208,10 +208,27 @@ do
         GROUP_ID=50
       fi
       docker build -t avro-build-${USER_NAME} - <<UserSpecificDocker
-FROM avro-build
+FROM docker.io/surajacharya/avro-build-test
```
Comment (busbey) [inline build.sh]: presumably this will be an image from the project before we start using this?

File: share/docker/Dockerfile [hunk 2]
```diff
@@ -17,12 +17,10 @@
 # Dockerfile for installing the necessary dependencies for building Avro.
 # See BUILD.txt.
 
-FROM java:7-jdk
+FROM docker.io/surajacharya/avro-build-test
```
Comment (busbey) [inline share/docker/Dockerfile]: presumably this will be an image from the project before we start using this?

File: build.sh (hunk 1 above)
Comment (spacharya) [inline build.sh]: Yes. Once there is a general consensus that the base image being used is good enough I intend to move it to a more central location (apache docker hub) and point it to that.

**Comment by spacharya** [comment]
Okay i will add the script to the patch shortly.

**Comment by busbey** [comment]
any update on including the example script?

**Comment by Fokko** [comment]
Superseded by #353

**Pull request closed without merging**